packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -838,7 +889,7 @@ export default function makeKernelKeeper(
       insistVatID(expectedDecider);
     }
     const p = getKernelPromise(kpid);
-    p.state === 'unresolved' || Fail`${kpid} was already resolved`;
+    assert.equal(p.state, 'unresolved', `${kpid} was already resolved`);
```

Comment:
These changes in particular (```x === y || Fail`…`;``` → ```assert.equal(x, y, `…`);```) don't seem like improvements to me.

Reply:
```
    if (p.state !== 'unresolved') {
      throw Fail`${kpid} was already resolved`;
    }
```
worked, the `foo || Fail` form causes TSC complaints that I don't know how to work around.